src/Open3D/GUI/Window.cpp

```diff
@@ -544,8 +676,21 @@ void Window::OnMouseEvent(const MouseEvent &e) {
             break;
         case MouseEvent::WHEEL: {
             ImGuiIO& io = ImGui::GetIO();
-            io.MouseWheelH += (e.wheel.dx > 0 ? 1 : -1);
-            io.MouseWheel  += (e.wheel.dy > 0 ? 1 : -1);
+            float dx = 0.0, dy = 0.0;
+            if (e.wheel.dx != 0) {
+                dx = e.wheel.dx / std::abs(e.wheel.dx);
+            }
+            if (e.wheel.dy != 0) {
+                dy = e.wheel.dy / std::abs(e.wheel.dy);
+            }
+            // Note: ImGUI's documentation says that 1 unit is about 5 lines
```

Comment:
What is 'unit' and what is 'line'? Just wondering, not sure it needs explanation in code

Reply:
1 unit of e.wheel.dy is 5 lines of text. I'll clarify the comment, since I'm sure you won't be the only one to ask :) And it does look a little unclear.